hedera-node/src/test/resources/testfiles/R4FeeSchedule.json

```diff
@@ -4,7 +4,8 @@
       {
         "transactionFeeSchedule": {
           "hederaFunctionality": "CryptoCreate",
-          "feeData": {
+          "feeData": [{
+            "subType": "Empty",
```

Comment:
Do we need "empty" here everywhere?

Reply:
when we start adding unit tests we can figure this out